engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/resolve/TypeSignatures.scala

```diff
@@ -322,7 +322,7 @@ case object TypeSignatures extends IRPass {
     *
     * @param signature the expression for the type signature
     */
-  case class Signature(signature: Expression) extends IRPass.IRMetadata {
+  case class Signature(signature: Expression, comment: Option[String] = None) extends IRPass.IRMetadata {
```

Comment:
This will likely change the cache serialization format - you'd rather find new number of the `@Persistable(Signature.class)`